samples/batch_fetch_test.rb

```diff
@@ -11,7 +11,8 @@
 )
 
 batch_fetch_request = {
-  :filters => "BatchId=2956681"
+  :fields => "Files",
+  :filters => "BatchId=3009916"
```

Comment:
Why the ID changes? Seems like these shouldn't really be part of the test file at all?